scripts/variantstore/wdl/GvsExtractCohortFromSampleNames.wdl

```diff
@@ -12,13 +12,15 @@ workflow GvsExtractCohortFromSampleNames {
     String query_project
     String gvs_project
     String gvs_dataset
+    String cohort_table_prefix
 
     # not using the defaults in GvsPrepareCallset because we're using pre created datasets defined by the caller
     String fq_gvs_extraction_destination_dataset
     String fq_gvs_extraction_temp_tables_dataset
 
     String extraction_uuid
     String? output_gcs_dir
+    String? service_account_json_path
```

Comment:
FYI I don't know anything about these changes, or the override changes, but I don't have objections